.github/workflows/wheels.yml

```diff
@@ -0,0 +1,125 @@
+# Workflow to build and test wheels
+name: Wheel builder
+
+on:
+  schedule:
+    # Nightly build at 3:42 A.M.
+    - cron: "42 3 */1 * *"
+  push:
+    tags:
+      # Final releases, release candidates and post-releases
+      - "[0-9]+.[0-9]+.[0-9]+(rc[0-9]+)?(.post[0-9]+)?"
+  pull_request:
+    branches:
+      - master
+      - "[0-9]+.[0-9]+.X"
+
+env:
+  SCIKIT_LEARN_VERSION: 0.24.dev0
+
+jobs:
+  # Check whether to build the wheels and the source tarball
+  check_build_trigger:
+    name: Check build trigger
+    runs-on: ubuntu-latest
+    outputs:
+      build: ${{ steps.check_build_trigger.outputs.build }}
+
+    steps:
+      - name: Checkout scikit-learn
+        uses: actions/checkout@v1
+
+      - id: check_build_trigger
+        name: Check build trigger
+        run: bash build_tools/github/check_build_trigger.sh
+
+  # Build the wheels for Linux, Windows and macOS for Python 3.6 and newer
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }} for Python ${{ matrix.python }}
+    runs-on: ${{ matrix.os }}
+    needs: check_build_trigger
+    if: needs.check_build_trigger.outputs.build
+
+    strategy:
+      # Ensure that a wheel builder finishes even if another fails
+      fail-fast: false
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        python: [36, 37, 38, 39]
+
+    steps:
+      - name: Checkout scikit-learn
+        uses: actions/checkout@v1
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+
+      - name: Build and test wheels
+        env:
+          # Set the directory where the wheel is unpacked
+          CIBW_ENVIRONMENT: "WHEEL_DIRNAME=scikit_learn-$SCIKIT_LEARN_VERSION"
```

Comment:
Isn't there a need of name conflict for the 32 bit and 64 bit windows builds?

Reply:
This is the default path for the `wheel` command (I just use an environment variable to make it cleaner `CIBW_REPAIR_WHEEL_COMMAND_WINDOWS`).

In fact, note that we remove this temporary directory just after moving the repaired wheel to the destination directory (note that this command is run after building each wheel). Therefore, I do not think that there is a naming conflict.

```shell
CIBW_REPAIR_WHEEL_COMMAND_WINDOWS: wheel unpack {wheel} &&
                                   python build_tools/github/vendor_vcomp140.py %WHEEL_DIRNAME% &&
                                   wheel pack %WHEEL_DIRNAME% -d {dest_dir} &&
                                   rmdir /s /q %WHEEL_DIRNAME%
```

Reply:
Indeed, you are right. Thanks for the clarification.